apps/api/src/domain/models/context/repositories/arango-note-query-repository.spec.ts

```diff
@@ -366,18 +375,21 @@ describe(`ArangoNoteQueryRepository`, () => {
             id: buildDummyUuid(101),
             type: WIDGET_TYPE,
             rating: 1,
+            tags: [],
         });
 
         const fromWidget = new WidgetViewModel({
             id: buildDummyUuid(102),
             type: WIDGET_TYPE,
             rating: 2,
+            tags: [],
```

Comment:
We might want to use our `buildTestInstance` within this file to make the test setup less repetitive.